app/views/prior_authority/steps/reason_why/edit.html.erb

```diff
@@ -0,0 +1,56 @@
+<%= javascript_import_module_tag 'file-upload' %>
+
+<% title t('.page_title') %>
+<% decision_step_header %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= govuk_error_summary(@form_object) %>
+    <span class="govuk-caption-xl"><%= t(".caption") %></span>
+    <%= step_form @form_object do |form| %>
+      <%= form.govuk_text_area :reason_why,  max_chars: 2000, rows: 16, label: { size: 'xl', text: t(".page_title") }, hint: { text: t('.reason_why_hint_html')}  %>
```

Comment:
Complex field hints are an absolute pain and I've not yet found a nice way of handling them. Putting HTML in the I18n yml is a decent enough workaround. One other thing you can do is:

```
hint: -> { content_for(:reason_why_hint) })
```

And then you can keep the HTML in your view by adding at the top:

```
<% content_for(:reason_why_hint) do %>
  <p>blah blah blah</p>
<% end %>
```
If you don't feel like that's any nicer then feel free to stick with what you've done.

Reply:
I want to have a chat about translations as we did it a different way for the crm7 work which was slightly nicer/easier.. will bring this up on our Thursday meeting.